vgg16_interpretation.ipynb

```diff
@@ -205,19 +225,33 @@
         "print(\"Visualizing image for class index \" + str(filter_idx) + \": \" + classidx_to_description_dict[filter_idx])\n",
         "img = visualize_activation(model, prediction_layer_index, filter_indices=[filter_idx], max_iter=100, tv_weight=1., lp_norm_weight=0.)\n",
         "ax2 = fig.add_subplot(2, 2, 1)\n",
-        "plt.imshow(img)"
+        "plt.imshow(img)\n",
+        "\n",
+        "# Restoring model to its original state\n",
+        "model.layers[prediction_layer_index].activation = activations.softmax\n",
+        "model = utils.apply_modifications(model)"
       ],
       "execution_count": 0,
       "outputs": []
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "JiOB7SsbupDn",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Step 5 Try modifying *filter_idx* in Step 4 above to explore the features learned by different filters in the prediction layer. </br> Refer Step 2 to get description for each filter index. "
```

Comment:
The question is step 4 where we asked them to visualize baseline also needs this info. So might want to combine these.